testonly/loglb/main.go

```diff
@@ -65,6 +66,7 @@ func newRandomLoadBalancer(serverCfg string) (*randomLoadBalancer, error) {
 	return &lb, nil
 }
 
+// TODO(somebody): This function name collides with a builtin function and should be renamed
```

Comment:
Although I think IntelliJ's warnings about identifiers clashing with package names are overkill, most of the renames are innocuous enough that they're worth having if they reduce your warning fatigue. 

But not this one :-).